lv_objx/lv_chart.c

```diff
@@ -21,6 +21,12 @@
 #define LV_CHART_VDIV_DEF   5
 #define LV_CHART_PNUM_DEF   10
 #define LV_CHART_AXIS_MARGIN 50
+#define LV_CHART_AXIS_TICK_LABEL_MAX_LEN 16
```

Comment:
This could potentially... you could just move it to `lv_conf_templ.h`. Wait for @kisvegabor's opinion though.

Reply:
I agree. It should be in `lv_conf_templ.h`